### PR TITLE
Add log for poxynode transform data and querynode segment insert

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: internal/datanode/flow_graph_insert_buffer_node.go [hunk 1]
```diff
@@ -210,6 +210,17 @@ func (ibNode *insertBufferNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
...
+		log.Debug("DataNode flow_graph_insert_buffer_node", zap.Any("Fields", collSchema.Fields))
+		var fieldIDs []int64
+		var fieldTypes []schemapb.DataType
+		for _, field := range collSchema.Fields {
+			fieldIDs = append(fieldIDs, field.FieldID)
+			fieldTypes = append(fieldTypes, field.DataType)
+		}
+
+		log.Debug("DataNode flow_graph_insert_buffer_node", zap.Any("FieldIDs", fieldIDs))
+		log.Debug("DataNode flow_graph_insert_buffer_node", zap.Any("fieldTypes", fieldTypes))
```
Comment (sunby) [inline internal/datanode/flow_graph_insert_buffer_node.go:222]: Why print `fieldIDs` and `fieldTypes` in two same log record?

File: internal/querynode/flow_graph_insert_node.go [hunk 2]
```diff
@@ -136,7 +136,15 @@ func (iNode *insertNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 }
 
 func (iNode *insertNode) insert(insertData *InsertData, segmentID int64, wg *sync.WaitGroup) {
+	log.Debug("QueryNode::iNode::insert", zap.Any("SegmentID", segmentID))
```
Comment (xige-16) [inline internal/querynode/flow_graph_insert_node.go:139]: redundant？

Comment (czs007) [reply internal/querynode/flow_graph_insert_node.go:139]: Change it next pr

File: internal/datanode/flow_graph_insert_buffer_node.go (hunk 1 above)
Comment (czs007) [reply internal/datanode/flow_graph_insert_buffer_node.go:222]: Change it next pr

**Pull request merged**